### DISCUSSION TITLE
跨链中所述的chainselector是什么？

### DISCUSSION THREAD

**Discussion by kugool-sy** [smartcontractkit/Web3_tutorial_Chinese#16] [Q&A]
跨链中所述的chainselector是什么？我一直以为是chainID，谁能帮忙解释一下吗？

**Comment by QingyangKong** [comment]
chainselector 是 CCIP 规定的的每个不同区块链的 ID，和 ChainID 不一样，chainId 是链自己定义的，而 chainselector 是由 Chainlink CCIP 决定的。可以在[官方文档](https://docs.chain.link/ccip/directory)中查看 CCIP 所支持的不同的区块链的 chainselector。

在使用 Chainlink CCIP 跨链的时候，需要告诉 CCIP 消息需要跨到哪条链，这时候就要使用 chainselector 来约定目标链。
